ods_ci/tests/Tests/1000__model_serving/1010__model_serving_triton_on_modelmesh/1010__model_serving_triton_on_modelmesh.robot

```diff
@@ -67,6 +71,43 @@ Test Onnx Model Rest Inference Via UI (Triton on Modelmesh)
     ...  project_title=${PRJ_TITLE}
     ...  AND
     ...  Clean All Models Of Current User
+    
+Test Pytorch Model Rest Inference Via UI (Triton on Modelmesh)
+    [Documentation]    Test the deployment of an onnx model in Kserve using Triton
+    [Tags]    Sanity    RHOAIENG-11561
+
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    ...    existing_project=${FALSE}
+    Open Dashboard Settings    settings_page=Serving runtimes
+    Upload Serving Runtime Template    runtime_filepath=${PYTORCH_RUNTIME_FILEPATH}
+    ...    serving_platform=multi      runtime_protocol=REST
+    Serving Runtime Template Should Be Listed    displayed_name=${ONNX_RUNTIME_NAME}
+    ...    serving_platform=multi
+    Recreate S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=model-serving-connection
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=ods-ci-s3
+    Create Model Server    token=${TRUE}    runtime=${ONNX_RUNTIME_NAME}    server_name=${ONNX_RUNTIME_NAME}    existing_server=${TRUE}
+    Sleep    10s
+    Serve Model    project_name=${PRJ_TITLE}    model_name=${ONNX_MODEL_NAME}    framework=pytorch - 1
+    ...    existing_data_connection=${TRUE}    data_connection_name=model-serving-connection
+    ...    model_path=triton/model_repository/resnet50/        model_server=${ONNX_RUNTIME_NAME}
+    Wait Until Runtime Pod Is Running    server_name=${ONNX_RUNTIME_NAME}
+    ...    project_title=${PRJ_TITLE}    timeout=5m
+    Verify Model Status    ${PYTORCH_MODEL_NAME}    success
+    ${EXPECTED_INFERENCE_REST_OUTPUT_PYTORCH}=     Load Json File      file_path=${EXPECTED_INFERENCE_REST_OUTPUT_FILE_PYTORCH}
+    ...     as_string=${TRUE}
+    Log     ${EXPECTED_INFERENCE_REST_OUTPUT_PYTORCH}
+    Verify Model Inference With Retries    ${PYTORCH_MODEL_NAME}    ${INFERENCE_REST_INPUT_PYTORCH_FILE}
+    ...    ${EXPECTED_INFERENCE_REST_OUTPUT_PYTORCH}
+    ...    token_auth=${TRUE}
+    ...    project_title=${PRJ_TITLE}
+    Open Dashboard Settings    settings_page=Serving runtimes
+    Delete Serving Runtime Template         displayed_name=modelmesh-triton
```

Comment:
maybe it should good to delete via CLI the serving runtime in the teardown in case something goes wrong in the test